### ISSUE TITLE
Cannot create a dataset

### ISSUE THREAD

**Issue by yevhenskakun** [chartbrew/chartbrew#218]
After I try to create a dataset for 1 second the creation window appears and then it immediately disappears and nothing can be done @razvanilin 

![image](https://github.com/chartbrew/chartbrew/assets/121243396/d7a61b24-9442-46dc-8d12-74a3d41956b1)

![image](https://github.com/chartbrew/chartbrew/assets/121243396/867b1001-78ce-4402-8b61-fcf1f469251d)

Error response [GET /team/5/datasets/new/dataRequests:]
```
{
    "name": "SequelizeDatabaseError",
    "parent": {
        "length": 109,
        "name": "error",
        "severity": "ERROR",
        "code": "22P02",
        "position": "920",
        "file": "numutils.c",
        "line": "320",
        "routine": "pg_strtoint32",
        "sql": "SELECT .... WHERE \"DataRequest\".\"dataset_id\" = 'new';"
    },
    "original": {
        "length": 109,
        "name": "error",
        "severity": "ERROR",
        "code": "22P02",
        "position": "920",
        "file": "numutils.c",
        "line": "320",
        "routine": "pg_strtoint32",
        "sql": "SELECT .... WHERE \"DataRequest\".\"dataset_id\" = 'new';"
    },
    "sql": "SELECT .... WHERE \"DataRequest\".\"dataset_id\" = 'new';",
    "parameters": {}
}
```

![image](https://github.com/chartbrew/chartbrew/assets/121243396/9420980c-6c89-4bd4-b9f4-f3d71ceb87f0)

Error response [GET /team/5/datasets/new]:
```
{
    "name": "SequelizeDatabaseError",
    "parent": {
        "length": 110,
        "name": "error",
        "severity": "ERROR",
        "code": "22P02",
        "position": "2506",
        "file": "numutils.c",
        "line": "320",
        "routine": "pg_strtoint32",
        "sql": "SELECT .... WHERE \"Dataset\".\"id\" = 'new';"
    },
    "original": {
        "length": 110,
        "name": "error",
        "severity": "ERROR",
        "code": "22P02",
        "position": "2506",
        "file": "numutils.c",
        "line": "320",
        "routine": "pg_strtoint32",
        "sql": "SELECT .... WHERE \"Dataset\".\"id\" = 'new';"
    },
    "sql": "SELECT .... WHERE \"Dataset\".\"id\" = 'new';",
    "parameters": {}
}
```

**Comment by yevhenskakun** [comment]
@razvanilin no i dont, every time when i try to save connection it shows window (pic.2) and after clicking on create a dataset i get the same error
![image](https://github.com/chartbrew/chartbrew/assets/121243396/803138ce-e045-4a85-b5b0-7d0f6742d468)

![image](https://github.com/chartbrew/chartbrew/assets/121243396/0972d58e-424b-4156-912f-c93233f93fd4)

**Comment by yevhenskakun** [comment]
for [POST /team/5/connections] i get this:
![image](https://github.com/chartbrew/chartbrew/assets/121243396/ee217fe5-f314-4576-849e-3d986d01e0a6)

**Comment by razvanilin** [comment]
The `CB_ENCRYPTION_KEY` or `CB_ENCRYPTION_KEY_DEV` (for development) will need a valid 32 bytes key as a value. You can generate one from here: https://generate-random.org/encryption-key-generator?count=1&bytes=32&cipher=aes-256-cbc&string=&password=

**Comment by yevhenskakun** [comment]
Same error, is my key wrong @razvanilin :

```
 - CB_ENCRYPTION_KEY=fV5OQNA4DemOlVTGm8VSHqXQZJBpv5aWP9PfRuGHXQTGlbAjxRIo/6w+wz+XJza3
 - CB_ENCRYPTION_KEY_DEV=f9aeamnXeZeN90gWtdaftRbKXYWkRpdkGYPSLBW6q/mN0xoFoXpnStBp83MrZbVo
```

**Comment by razvanilin** [comment]
I just ran some tests with those and they are not valid. I think the website that I quoted is a bit of a hit and miss with the results, apologies for recommending that!

You can instead use `openssl` to generate a key like so:

```
openssl rand -hex 32
```

What version are you running btw? I just applied an important fix in 3.4.0 and you should upgrade if you haven't done so yet

**Comment by yevhenskakun** [comment]
i use 3.4.0

**Comment by yevhenskakun** [comment]
> I just ran some tests with those and they are not valid. I think the website that I quoted is a bit of a hit and miss with the results, apologies for recommending that!
> 
> You can instead use `openssl` to generate a key like so:
> 
> ```
> openssl rand -hex 32
> ```
> 
> What version are you running btw? I just applied an important fix in 3.4.0 and you should upgrade if you haven't done so yet

It works !!
